engine-rest/engine-rest-openapi/src/main/templates/paths/external-task/fetchAndLock/post.ftl

```diff
@@ -18,6 +18,10 @@
                          "workerId": "aWorkerId",
                          "maxTasks": 2,
                          "usePriority": true,
+                         "createTimeConfig":{
+                            "useCreateTime":"false",
+                            "direction":"ASC"
+                         },
```

Comment:
❌ Does this example make sense with `useCreateTime = false`? Isn't it the same as if this config is not applied?

Reply:
Yeap, true. I've replaced the `createTimeConfig` with the new `sorting` array field which now will include `createTime` as a sorting option for the Open API as well.